### PR TITLE
LPS-102299 CCR

### PR DESCRIPTION
**Pull request by BryanEngler**
Hey Russ, can you take a look at the lang keys and .bnd descriptions etc. Thanks!

### CODE REVIEW

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -10,8 +10,11 @@ bootstrap-mlockall=Bootstrap Mlockall
 bootstrap-mlockall-help=This configuration only applies to EMBEDDED mode. If this is checked, the process address space is locked into RAM, preventing any Elasticsearch memory from being swapped out.
```
Comment (BryanEngler) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-api/src/main/resources/content/Language.properties]: here

File: modules/dxp/apps/portal-search-elasticsearch-cross-cluster-replication/app.bnd [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Liferay-Releng-App-Description: %liferay-releng-app-description
```
Comment (BryanEngler) [inline modules/dxp/apps/portal-search-elasticsearch-cross-cluster-replication/app.bnd]: here

File: modules/dxp/apps/portal-search-elasticsearch-cross-cluster-replication/portal-search-elasticsearch-cross-cluster-replication-impl/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+cross-cluster-replication-configuration-name=Cross-Cluster Replication
```
Comment (BryanEngler) [inline modules/dxp/apps/portal-search-elasticsearch-cross-cluster-replication/portal-search-elasticsearch-cross-cluster-replication-impl/src/main/resources/content/Language.properties]: here

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -0,0 +1,3 @@
+please-set-a-value-for-network-host-addresses=Please set a value for Network Host Addresses.
```
Comment (BryanEngler) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties]: here

Comment (rbohl) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties]: @bryanengler, where does this one appear? Doesn't look like a configuration lang key.

Comment (rbohl) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties]: Maybe an error message?

Comment (BryanEngler) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties]: https://github.com/rbohl/liferay-portal/pull/24/files#diff-7257252dbfdb6fae9a756f9586b715db

Comment (BryanEngler) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties]: ElasticsearchConnectionConfigurationModelListener

Comment (BryanEngler) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties]: but ya, its basically an error message

Comment (rbohl) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties]: ahh, okay. i think i like this better assuming it's accurate:
> Please set at least one network host address.

Comment (rbohl) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/resources/content/Language.properties]: Do we have duplicate keys in the -api and -impl modules? Is that necessary?

**Comment by rbohl** [comment]
Superseded: https://github.com/BryanEngler/liferay-portal/pull/454

**Pull request closed without merging**